### PR TITLE
Provides minor improvements for the metrics framework.

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta [hunk 1]
```diff
@@ -1,29 +1,30 @@
 <i:arg type="java.lang.Object" name="target"/>
 
-<i:pragma name="description">Renders a sindebar view of some key metrics for the given target as supplied by "MetricDashboard"</i:pragma>
+<i:pragma name="description">Renders a sindebar view of some key metrics for the given target as supplied by
+    "MetricDashboard"
+</i:pragma>
```
Comment (sabieber) [inline src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta:5]: This doesnt break Tagliatelle?

**Review by sabieber — APPROVED** [review]

File: src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta (hunk 1 above)
Comment (andyHa) [reply src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta:5]: grafik
Nope...

Comment (sabieber) [reply src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta]: K, then it just looks strange :D

Comment (sabieber) [reply src/main/resources/default/taglib/t/sidebarKeyMetrics.html.pasta]: K, then it just looks strange :D

**Pull request merged**